src/components/RelatiesKoppelingenTekstueel/RelatiesKoppelingenTekstueel.js

```diff
@@ -9,6 +9,16 @@ const getObjectFromRelation = (relation) => {
         : relation
 }
 
+/**
+ * Component that renders the RelatiesKoppelingenTekstueel component,
+ *
+ * @component
+ *
+ * @param {object} beleidsObject - Parameter containing the information of a beleid
```

Comment:
The 'Parameter containing' part can be left out, as it is clear we that we are talking about a parameter